src/Tizen.Network.NetworkMonitor/Tizen.Network.NetworkMonitor/NetworkMonitor.cs

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.Network.NetworkMonitor
+{
+    /// <summary>
+    /// NetworkMonitor class provides functions to get information about connections and links.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public static class NetworkMonitor
```

Comment:
What is difference from `ConnectionManager`?

Reply:
Actually, many properties and methods of NetworkMonitor have the same functions as those of Tizen.Network.Connection and Tizen.Network.WiFi.
(It is the corresponding C# API of capi-network-inm which contains the functions although a lot of them are already provided in capi-network-connection and capi-network-wifi)
For duplicate functions, can we add APIs in other namespace?

Reply:
I don't understand, why this api should be added. Why need to add NetworkMonitor moudle

Reply:
NetworkMonitor contains many duplicate APIs. 
If C# already have a lot of checker that we'll add in NetworkMonitor and duplicate APIs should not be added, we'll review again whether NetworkMonitor is a necessary module or not.
Thanks.

Reply:
There are new APIs that other modules don't provide: Network interface information (Link class) and route information (Route class).
If there is a way to obtain those information, we don't have to add NetworkMonitor.

Reply:
NetworkInterface class: https://docs.microsoft.com/ko-kr/dotnet/api/system.net.networkinformation.networkinterface?view=netframework-4.8